### ISSUE TITLE
App crashes on startup on iOS 12.4

### ISSUE THREAD

**Issue by rist** [Flipboard/FLEX#379]
we are using FLEX 3.1.2 and our app crashes immediately after startup on iOS 12.x simulator and devices.

The crash happens at FLEXNetworkObserver.m:183 with ```Thread 1: signal SIGABRT```

**Comment by rist** [comment]
we found the root cause in our code and a fix - not sure if there is anything to be improved in FLEX

we had:

```
@available(iOS 13.0, *)
public class OverviewViewController: UIViewController {		

    lazy var dataSource: UICollectionViewDiffableDataSource<OverviewSection, UIColor> = …

}
```

we got rid of the crash by replacing that lazy property with a method:
```private func createDataSource() -> UICollectionViewDiffableDataSource<OverviewSection, UIColor> {```

**Comment by NSExceptional** [comment]
If you can post a crash log or a stack trace I can still take a look
